src/main/resources/assets/computercraft/lua/rom/apis/paintutils.lua

```diff
@@ -12,8 +12,8 @@ end
 
 local tColourLookup = {}
 for n=1,16 do
-    tColourLookup[ string.sub( "0123456789abcdef",n,n ) ] = 2^(n-1)
-    tColourLookup[ 2^(n-1) ] = string.sub( "0123456789abcdef",n,n )
+    tColourLookup[ string.byte( "0123456789abcdef",n,n ) ] = 2^(n-1)
+    tColourLookup[ 2^(n-1) ] = string.byte( "0123456789abcdef",n,n )
```

Comment:
If you want to do this, then later in the code you'll also need to change some "table.concat"s into "string.char( unpack () )"s. I'm fairly sure it's faster just to use string.sub().